llvm/lib/IR/DebugInfo.cpp

```diff
@@ -375,6 +375,32 @@ bool DebugInfoFinder::addScope(DIScope *Scope) {
   return true;
 }
 
+// Recursively handle DILocations in followup metadata etc.
```

Comment:
```suggestion
/// Recursively handle DILocations in followup metadata etc.
```
[nit] doxygen comment for global function documentation

Reply:
Fixed in the latest update of this PR.